packages/@aws-cdk/aws-lambda-nodejs/test/integ.function.ts

```diff
@@ -29,6 +29,28 @@ class TestStack extends Stack {
       runtime: Runtime.NODEJS_14_X,
       vpc: new Vpc(this, 'Vpc'),
     });
+
+    new lambda.NodejsFunction(this, 'ts-handler-custom-handler-no-dots', {
+      entry: path.join(__dirname, 'integ-handlers/ts-handler.ts'),
+      runtime: Runtime.NODEJS_14_X,
+      bundling: {
+        minify: true,
+        sourceMap: true,
+        sourceMapMode: lambda.SourceMapMode.BOTH,
+      },
+      handler: 'someHandler',
+    });
+
+    new lambda.NodejsFunction(this, 'ts-handler-custom-handler-dots', {
+      entry: path.join(__dirname, 'integ-handlers/ts-handler.ts'),
+      runtime: Runtime.NODEJS_14_X,
+      bundling: {
+        minify: true,
+        sourceMap: true,
+        sourceMapMode: lambda.SourceMapMode.BOTH,
+      },
+      handler: 'run.sh',
```

Comment:
Is there a way we can make this example not only deploy but actually function if invoked? `run.sh` doesn't exist in this directory, but it seems like a good place to add one of the lambda extensions you referenced

Reply:
Thanks, yeah that is simple enough.

Reply:
I have added code that makes the handler override work (well... it doesn't setup a Function URL or API Gateway for the Lambda, but if it did, it should work) and confirmed it deploys.